### PR TITLE
Follow shaman jump in waterfall

### PR DESCRIPTION
**Pull request by jayant-tripathi**
Please review this, @casey-collab

### CODE REVIEW

File: english/forest/fire/woods/follow/jumpInWaterFall/jump.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+You fall and keep falling until you hit the water hard. You feel pain as you have never experienced before.
```
Comment (casey-collab) [inline english/forest/fire/woods/follow/jumpInWaterFall/jump.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/forest/fire/woods/follow/jumpInWaterFall/notJump.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Your are killed by bear while stung by bees.
```
Comment (casey-collab) [inline english/forest/fire/woods/follow/jumpInWaterFall/notJump.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/forest/fire/woods/follow/jumpInWaterFall/jump.md (hunk 1 above)
Comment (jayant-tripathi) [inline english/forest/fire/woods/follow/jumpInWaterFall/jump.md]: Please review this better now?, @casey-collab

File: english/forest/fire/woods/follow/jumpInWaterFall/notJump.md (hunk 2 above)
Comment (jayant-tripathi) [inline english/forest/fire/woods/follow/jumpInWaterFall/notJump.md]: Please review this better now?, @casey-collab

**Comment by jayant-tripathi** [comment]
Please review this better now?, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :octocat:

**Pull request merged**